client/state/selectors/get-directly-config.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Get the configuration options used when initializing the Directly library
+ *
+ * @param {Object} state Global state tree
+ * @return {Object} The configuration options used by Directly
+ */
+export default function getDirectlyConfig( state ) {
+	return state.directly.config;
```

Comment:
we can be sensitive to the fact that this is an _unsafe_ operation when `directly` doesn't exist. using a safe-defaulting getter could remove the risk…

```js
export const getDirectlyConfig = state => get( state, 'directly.config' );
```